x-pack/plugins/observability_solution/observability/public/navigation_tree.ts

```diff
@@ -83,6 +67,9 @@ export function createNavTree(pluginsStart: ObservabilityPublicPluginsStart) {
           {
             link: 'slo',
           },
+          {
+            link: 'inventory',
```

Comment:
I found that the navigation tree affecting Serverless is located in `kibana/x-pack/plugins/serverless_observability/public/navigation_tree.ts`. I still need to determine how the one under `/observability/` (the one you modified) impacts the system.

We can keep this in mind and I'll handle it in my [issue](https://github.com/elastic/kibana/issues/192323). If we decide to leave it here, I recommend keeping it under the `pluginsStart.inventory` flag until we update it with the `observability:entityCentricExperience` advanced setting.

Reply:
This shouldn’t block the PR from being merged, but I believe there may be a misunderstanding.

Removing the Inventory item from the `xpack/plugins/observability_solution/observability/public/navigation_tree.ts` doesn’t seem to impact the UI. 

The Inventory is still visible in the side navigation simply by registering the plugin the way you’ve done.

Reply:
`xpack/plugins/observability_solution/observability/public/navigation_tree.ts` 

This file is not used for the **stateful classic** view that you're shared the screenshot . 

- navigation_tree affects the **stateful observability** and probably serverless (I haven't verified this one_

Reply:
As I mentioned earlier, Serverless seems to be impacted by `kibana/x-pack/plugins/serverless_observability/public/navigation_tree.ts`, not `kibana/x-pack/plugins/observability/public/navigation_tree.ts`.

Regarding the stateful observability sidebar, I’ll reach out to you privately since it seems like I’m missing some context on that.

Thanks!

Reply:
@iblancof Feel free to ping me as well to clarify the confusion, since I am working on a o11y navigation related issue. We have this well documented [issue](https://github.com/elastic/kibana/issues/192050), where the three different types of navigation are explained. 

- stateful classic
- stateful observability (defined as stateful cloud in our issue)
- serverless

@iblancof To your question, yep the new stateful observability sidebar is the one you get when configuring a space. You can check our issue with the flags you need to add to your kibana.dev.yml as well and the change you need to do under `Stack Management > Spaces` to activate this view. This new o11y navigation is controlled by the file you listed `xpack/plugins/observability_solution/observability/public/navigation_tree.ts`

@kpatticha I confirm that serverless nav is controlled by `https://github.com/elastic/kibana/blob/main/x-pack/plugins/serverless_observability/public/navigation_tree.ts` as @iblancof mentioned above

Reply:
Hi @mgiota, 

Thank you for providing all the context!

As you mentioned, after experimenting with the code a bit, we were able to fully understand how everything works. It all makes sense to us now. Thanks again for your help!